public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -290,7 +291,7 @@ function register_rest_fields() {
 		'favorite_count',
 		array(
 			'get_callback' => function() {
-				return 1;
+				return get_favorite_count( get_the_ID() );
 			},
 
 			'schema' => array(
```

Comment:
Would it be worth adding a `'default' => 0` to the schema here, so that there's always a value for this field even if there are no favorites?

Reply:
`get_favorite_count` will always return an int, even if there are no favorites, but I'll add it.